validation.py

```
@@ -0,0 +1,681 @@
+            sanitized["task_requirements"] = task_req
+        
+        # Sanitize brand_bible
+        if "brand_bible" in sanitized and sanitized["brand_bible"]:
+            brand_bible = sanitized["brand_bible"].copy()
+            if "xml_raw" in brand_bible:
+                brand_bible["xml_raw"] = self._sanitize_xml(brand_bible["xml_raw"])
+            sanitized["brand_bible"] = brand_bible
+        
+        return sanitized
+    
+    def _normalize_platforms(self, platforms: List[str]) -> List[str]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)